src/main/java/gift/auth/security/JwtFilter.java

```diff
@@ -0,0 +1,63 @@
+package gift.auth.security;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+@Component
+public class JwtFilter extends OncePerRequestFilter  {
+
+    private final JwtUtil jwtUtil;
+
+    public JwtFilter(JwtUtil jwtUtil) {
+        this.jwtUtil = jwtUtil;
+    }
+
+    @Override
+    protected boolean shouldNotFilter(HttpServletRequest request) throws ServletException {
+        String[] excludePath = {"/members/register", "/members/login"};
+        String path = request.getRequestURI();
+        return Arrays
+                .stream(excludePath)
+                .anyMatch(path::startsWith);
+    }
+
+    @Override
+    protected void doFilterInternal(
+            HttpServletRequest request,
+            HttpServletResponse response,
+            FilterChain filterChain) throws ServletException, IOException {
+
+        String authHeader = request.getHeader("Authorization");
+
+        if (authHeader == null) {
+            response.sendError(HttpStatus.UNAUTHORIZED.value(), "인증이 유효하지 않습니다.");
+            return;
+        }
+
+        if (!authHeader.startsWith("Bearer ")) {
```

Comment:
이 부분도 상수로 관리함으로써 실수로 문자열을 잘못 입력하는 것을 방지할 수 있을 것 같아요!

```suggestion
public static final String BEARER_PREFIX = "Bearer ";

        if (!authHeader.startsWith(BEARER_PREFIX)) {
```

Reply:
해당 문자열과 문자열 길이 모두 상수로 정의해놓겠습니다!